### DISCUSSION TITLE
Having troube running benchmarks

### DISCUSSION THREAD

**Discussion by viceice** [sebastienros/jint#1773] [Q&A] [answered]
I'm trying to run the benchmarks, especially the engine comparision. i like to add clearscript v8 to compare.

I'm always running into same issue: 

```
Did not find ~\jint\artifacts\bin\Jint.Benchmark\release\27eb5792-3361-461e-ba66-1cee2793d182\bin\Release\net8.0\27eb5792-3361-461e-ba66-1cee2793d182.dll, but the folder contained:
27eb5792-3361-461e-ba66-1cee2793d182.dll.config
Executable ~\jint\artifacts\bin\Jint.Benchmark\release\27eb5792-3361-461e-ba66-1cee2793d182\bin\Release\net8.0\27eb5792-3361-461e-ba66-1cee2793d182.dll not found
ExitCode != 0 and no results reported
No Workload Results were obtained from the run.
```

Any idea? This happens on all tests

**Comment by viceice** [comment]
this was running on windows, but i also tried to run on wsl2 with same issue.

**Comment by lahma** [comment, marked as answer]
Can you update to latest `main`, I did a small tweak.

**Reply by viceice** [reply]
- #1774

Thanks, works.

**Comment by viceice** [comment]
It seems `UseArtifactsOutput` is causing the issue, commenting it out allows the Benchmark to run

- https://github.com/sebastienros/jint/pull/1689
- https://github.com/dotnet/BenchmarkDotNet/issues/2466


`UseArtifactsOutput` will be supported when v0.14.0 of BenchmarkDotNet will be released
